CollAction/CollAction.csproj

```diff
@@ -4,7 +4,33 @@
     <UserSecretsId>aspnet-CollAction-29a5d7b8-3530-4482-9ed6-572bd6c178fa</UserSecretsId>
     <LangVersion>8.0</LangVersion>
     <Nullable>enable</Nullable>
+    <PreserveCompilationReferences>true</PreserveCompilationReferences>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
Why specifically remove them, and add them below here? Probably something VS did?

Reply:
This is what happens if you add them as embeddedresource (which is what razorlight uses). Pretty sure what visual studio did is correct, bit weird, but if you remove parts it doesn't work.